app/Models/Prescription.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class Prescription extends Model
```

Comment:
I'm not sure if you have the command available, but if you do, you can run `sail php artisan ide-helper:models` to have all the description of the models available to have a code with more navigation in it.